bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/utils/Messages.java

```diff
@@ -334,6 +335,7 @@ public class Messages extends NLS {
 	public static String WM_mutexAbandoned;
 
 	public static String updateUnknownNatureMarkers;
+	public static String updateMissingProjectEncodingMarkers;
```

Comment:
This is not used anymore, please remove

Reply:
removed line (=> no further changes => this class is no longer listed in the PR)